deepchem/models/tensorgraph/models/text_cnn.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python2
```

Comment:
Maybe remove this line? This file probably shouldn't be command line executable.